Solutions/Marain.Tenancy.Host.Functions/local.settings.template.json

```diff
@@ -8,8 +8,10 @@
     "APPINSIGHTS_INSTRUMENTATIONKEY": "",
     "AI:DeveloperMode": "true",
 
-    "TenantCloudBlobContainerFactoryOptions__RootTenantBlobStorageConfiguration__AccountName": "martenancyprod",
-    "TenantCloudBlobContainerFactoryOptions__RootTenantBlobStorageConfiguration__KeyVaultName": "martenancyprod",
-    "TenantCloudBlobContainerFactoryOptions__RootTenantBlobStorageConfiguration__AccountKeySecretName": "TenancyAccountKey"
+    "TenantCloudBlobContainerFactoryOptions__AzureServicesAuthConnectionString": "",
+
+    "RootTenantBlobStorageConfigurationOptions__AccountName": "martenancyprod",
```

Comment:
Is this a suitable choice for local dev? I actually set this to `"RootTenantBlobStorageConfigurationOptions__AccountName": ""` to get it working with the local storage emulator on my system.